_docs/sysadmin/system_customization.md

```diff
@@ -8,8 +8,21 @@ order: 6
 _The optional instructions below are suggestions for the system
 administrators of a live Submitty installation._
 
-
-### Customize upload students script
+## <a name="toc">Table of Contents</a>
+* [Customize upload students script](#setcsvfields)
+* [Schedule backups of production server data](#schedule_backups)
+* [Capture cron error messages](#capture_cron)
+* [Configure log rotation](#log_rotation)
+* [Set password policy](#password_policy)
+* [Secure SSH](#secure_ssh)
+* [Block some brute-force ssh connections](#block_brute_force)
+* [Disable PHP Functions](#disable_php_functions)
+* [Allowing Large Student File Upload Submissions](#large_submissions)
+* [Show system message to all users](#system_messages)
+* [Adding Additional Links To The Footer](#footer_links)
```

Comment:
All of the headers on the page have a link already and we should probably use that. each link is just the string lowercased with hyphens instead of spaces. E.g. the TOC would be `[Back to Table of Contents](#table-of-contents)`